fe/fe-core/src/main/java/com/starrocks/sql/analyzer/Field.java

```diff
@@ -55,7 +55,7 @@ public Field(String name, Type type, TableName relationAlias, Expr originExpress
         this.relationAlias = relationAlias;
         this.originExpression = originExpression;
         this.visible = visible;
-        this.isNullable = false;
```

Comment:
Are you sure modify this not effect other logical?

Reply:
I add this field. the original default is true.